src/calculateRentalCost.js

```diff
@@ -29,6 +29,20 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const price = 40;
+  const smallDiscount = 20;
+  const bigDiscount = 50;
+  let total;
+
+  if (days >= 7) {
+    total = days * price - bigDiscount;
+  } else if (days >= 3) {
+    total = days * price - smallDiscount;
+  } else {
+    total = days * price;
+  }
+
+  return total;
```

Comment:
1) try to avoid using `else if` to make your code clearer. 
You can just use `early return`
2) also to avoid duplicate you can move `days * price` to a variable like `rentalCost = days * price`
```suggestion
  if (days >= 7) {
    return rentalCost - bigDiscount;
  } 
  
  if (days >= 3) {
    return rentalCost - smallDiscount;
  } 
  
  return rentalCost;
```